ReactNative/ReactTwitter/__tests__/core/service/oauth-helper.test.js

```diff
@@ -0,0 +1,41 @@
+jest.unmock('../../../core/service/oauth-helper.js')
+jest.unmock('../../../core/service/percent-encoder.js')
+// var OauthHelper = require('../../../core/service/oauth-helper.js')
+
+describe('OauthHelper', () => {
+
+  // commented out because it's failing for a strange reason
```

Comment:
We can pair tomorrow if you need some help on this (should be pretty straightforward).